integration/template/values.go

```diff
@@ -0,0 +1,53 @@
+package template
```

Comment:
`template.Values()` somehow doesn't connect with `values.yaml` for me. What if this package was called `values` and `Values()` was `YAML()`?

The template itself below is naturally `ValuesTemplate` which would kind of support `values` package and it could potentially be renamed to just `Template` even.

The logic related to text template rendering is often so small that I wouldn't maybe name a package after it when there's no other major logic related to templating.

WDYT?

Reply:
Done PTAL